udata_gouvfr/tests.py

```diff
@@ -341,6 +341,21 @@ def test_render_with_data(self):
         self.assert200(response)
 
 
+class SpdTest(FrontTestCase):
+    settings = GouvFrSettings
+
+    def test_render_without_data(self):
+        response = self.client.get(url_for('gouvfr.spd'))
+        self.assert200(response)
+
+    def test_render_with_data(self):
+        for i in range(3):
```

Comment:
One no, at least two to test a list iteration.
I'm used to define 3 elements (this is the cas in every listing test), but I can set it to 2.